doc/fd.1

```diff
@@ -159,9 +159,20 @@ can be used as an alias.
 Enable the display of filesystem errors for situations such as insufficient
 permissions or dead symlinks.
 .TP
-.B \-\-strip-cwd-prefix
-By default, relative paths are prefixed with './' when the output goes to a non interactive terminal
-(TTY). Use this flag to disable this behaviour.
+.B \-\-strip-cwd-prefix [when]
+By default, relative paths are prefixed with './' when -x/--exec,
+-X/--exec-batch, or -0/--print0 are given, to reduce the risk of a
+path starting with '-' being treated as a command line option. Use
+this flag to change this behavior. If htis flag is used without a value,
+it is equivalent to passing "always". Possible values are:
```

Comment:
I would have expected "auto" here, since it is described as the default behavior below?

Reply:
Making it equivalent to always makes it simpler to turn it on.  And I think that having `--strip-cwd-prefix` without a specific when value meaning "auto" instead of "always" could be confusing since it reals like it is always stripping the prefix.

Reply:
Also, using `always` if  this is used without an argument is more backwards compatible with existing usage of the argument.

Reply:
I think the thing that confused me was that "auto" is described as "Use the default behavior" below. And I misinterpreted that as "auto = do whatever you do when no argument is given". After re-reading everything, it still seems a bit strange to me, but I agree that `always` makes sense as a default.